### PR TITLE
docs: enable BlazorWebAssemblyLazyLoad

### PR DESCRIPTION
**Pull request by 239573049**
# Description

_Please explain the changes you've made_

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [ ] Created/updated tests
* [x] Extended the documentation

### CODE REVIEW

File: src/Docs/Masa.Docs.Shared/App.razor [hunk 1]
```diff
@@ -41,6 +41,18 @@
                     new[] { "Util.Reflection.dll" });
                 lazyLoadedAssemblies.AddRange(assemblies);
             }
+            else if (args.Path.StartsWith("stack"))
```
Comment (capdiem) [inline src/Docs/Masa.Docs.Shared/App.razor:44]: 1. 首页也需要加载 所有的docs.dll
2. 上面的两个组件的path错了，需要加 `blazor/` 前缀

File: src/Docs/Masa.Docs.WebAssembly/Masa.Docs.WebAssembly.csproj [hunk 2]
```diff
@@ -15,4 +15,12 @@
...
   </ItemGroup>
 
+	<ItemGroup>
+		<BlazorWebAssemblyLazyLoad Include="SkiaSharp.dll" />
+		<BlazorWebAssemblyLazyLoad Include="SkiaSharp.Views.Blazor.dll" />
+		<BlazorWebAssemblyLazyLoad Include="Util.Reflection.dll" />
+		<BlazorWebAssemblyLazyLoad Include="Masa.Stack.Docs.dll" />
+		<BlazorWebAssemblyLazyLoad Include="Masa.Framework.Docs.dll" />
+	</ItemGroup>
+
```
Comment (capdiem) [inline src/Docs/Masa.Docs.WebAssembly/Masa.Docs.WebAssembly.csproj:25]: 缩进不对。

**Review by capdiem — CHANGES_REQUESTED** [review]

**Comment by capdiem** [comment]
PR 标题改正确 and english please

File: src/Docs/Masa.Docs.WebAssembly/Masa.Docs.WebAssembly.csproj (hunk 2 above)
Comment (239573049) [reply src/Docs/Masa.Docs.WebAssembly/Masa.Docs.WebAssembly.csproj:25]: 看着没问题啊

**Review by capdiem — APPROVED** [review]
LGTM

**Pull request merged**